core/src/main/kotlin/services/OrchestratorService.kt

```diff
@@ -57,6 +57,7 @@ class OrchestratorService(
         labels: Map<String, String>?
     ): OrtRun {
         val ortRun = db.dbQuery(transactionIsolation = Connection.TRANSACTION_SERIALIZABLE) {
+            maxAttempts = 20
```

Comment:
Just curious, how did you come to this value of 20? Did you do some experiments? I would be interested in the effect it has to increase this value. For instance, if going from 3 repetitions to 5 or maybe 10, how many ORT runs could then be started in parallel?

On a side note, if somebody wants to start 50 parallel runs on a single repository, there should probably some rate limiting be applied.

Reply:
It's a bit random, I chose 50 for the test case to use a value so high that we will probably never see it in production. With 10 max attempts that still failed for a few of them, so I went with 20 to be on the safe side.

Reply:
Thanks for explaining.